mollie/organizations.go

```diff
@@ -0,0 +1,74 @@
+package mollie
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// Organization describes a organization detail
```

Comment:
Just some quick fix as go report card tracks grammar and misspelling
```suggestion
// Organization describes an organization detail
```